translator/config/envconst.go

```diff
@@ -8,6 +8,8 @@ const (
 	RUN_IN_CONTAINER_TRUE   = "True"
 	USE_DEFAULT_CONFIG      = "USE_DEFAULT_CONFIG"
 	USE_DEFAULT_CONFIG_TRUE = "True"
+	FORCE_MODE_EC2          = "FORCE_MODE_EC2"
```

Comment:
Could you change the name to something like "RUN_IN_AWS"? Currently the ec2 mode is overloaded with many meanings (e.g. ec2 instance, container running in ec2 instance in ecs, container running in ec2 instance in eks, container running in ecs fargate). We would like to do some refactoring about it. So I guess "RUN_IN_AWS" is probably more meaningful in you case.

Reply:
renamed the env name to `RUN_IN_AWS`